Formula/qt.rb

```diff
@@ -144,8 +144,11 @@ class Qt < Formula
     directory "qtwebengine"
   end
 
+  # Fix build in Python 3.11 (invalid mode: 'rU')
+  patch :DATA
```

Comment:
Current patch won't work as is given the submodule/packaging structure of Qt. Your patch is being applied to files `PRESUBMIT_test_mocks.py` and `tools/grit/grit/util.py` in full Qt source. The patch needs to be done in directory for Chromium code.

Also, it would be better to use upstream commit if it cleanly applies. GitHub mirror is usually easier to use similar to previous patch. This would be https://github.com/qt/qtwebengine-chromium/commit/55eb07dfc2498ea079998e6d3e6f9f38180206cb for 102-based backport.

Not sure where Qt lists current Chromium version (102-based vs 106-based branches), but v6.4.1 qtwebengine (https://github.com/qt/qtwebengine/tree/v6.4.1/src) shows submodule is on commit https://github.com/qt/qtwebengine-chromium/commit/9457651ead9b7034edb37532c2f33a558b1700b6. This appears to be in 102-based branch.